src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -25,45 +30,103 @@ export class NewsLayout extends Component<Props, State> {
 
     this.state = {
       isTruncated: this.props.isTruncated,
+      isHovered: this.props.isHovered,
     }
   }
 
   render() {
-    const { article } = this.props
-    const { isTruncated } = this.state
+    const { article, isMobile } = this.props
+    const { isTruncated, isHovered } = this.state
 
     return (
       <NewsContainer
-        onClick={() =>
-          this.setState({
-            isTruncated: false,
-          })}
         isTruncated={isTruncated}
```

Comment:
Is it intended that the entire box does not expand on click in desktop?

Reply:
IMO would be more user friendly to keep in fullscreen, and use an isMobile flag like below.